ui/packages/dashboard_components/src/EllipsisText/index.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react'
+import cx from 'classnames'
+import styles from './index.module.less'
+
+export interface IEllipsisTextProps
+  extends React.HTMLAttributes<HTMLDivElement> {
+  text: string
+}
+
+export default function EllipsisText({
+  text,
+  className,
+  ...rest
+}: IEllipsisTextProps) {
+  return (
+    <div className={cx(styles.ellipsis_text, className)} {...rest}>
+      {text}
```

Comment:
I discovered that we can actually use https://ant.design/components/typography/#Typography.Text

Reply:
awesome, let me have a try!

Reply:
unfortunately, didn't get the expected result:

![image](https://user-images.githubusercontent.com/1284531/79344563-931e9900-7f62-11ea-9618-932c293dee8b.png)

Reply:
Sad news!